test/test.c

```diff
@@ -31470,6 +31470,140 @@ void test_KeyValueKeys(void)
     JS_TEARDOWN;
 }
 
+void test_KeyValueKeysWithFilters(void)
+{
+    natsStatus          s;
+    kvStore             *kv = NULL;
+    kvKeysList          l;
+    char                *k = NULL;
+    kvConfig            kvc;
+    kvWatchOptions      o;
+    int                 i;
+    const char **defaultSubject = (const char *[]){">"};
+    l.Count = 0;
+    l.Keys = NULL;
+
+    JS_SETUP(2, 10, 14);
```

Comment:
Not sure we need that recent of a server version for this, do we?

Reply:
Well, it seems we do need that late of a version for one of the tests (multiple overlapping filters).